### PR TITLE
CA-386235, CA-386236, CA-386876  Fix some documents issues

### PR DESCRIPTION
**Pull request by catherineliuchina**

### CODE REVIEW

File: docs/xenserver-automated-certification-kit-guide.md [hunk 1]
```diff
@@ -191,13 +191,13 @@ For the automated certification kit to run successfully, there are currently the
...
 
-    xe update-upload file-name=”/root/xenserver-server-cert-kit-xs8.iso”   
+    xe update-upload file-name=/root/xenserver-server-cert-kit-xs8.iso 
 
 The command returns the update UUID of server certification kit package on successful upload.
 
     xe update-apply uuid=<update uuid of server certification kit> --multiple  
 
-Upload "vpx-dlvm.xva” file to the server certification kit home folder on all hosts "/opt/xensource/packages/files/auto-cert-kit" .
+Upload "vpx-dlvm.xva” file to the server certification kit home folder on **all hosts** "/opt/xensource/packages/files/auto-cert-kit" .
```
Comment (Zhengchai) [inline docs/xenserver-automated-certification-kit-guide.md:200]: where is the ""vpx-dlvm.xva” for uploading? 
I didn't see the steps mentioned this here?

Comment (catherineliuchina) [reply docs/xenserver-automated-certification-kit-guide.md:200]: How to create "vpx-dlvm.xva" has been described  in the "XenServer Installation" section. So the operator should know where the "XenServer Installation" is.

File: docs/xenserver-automated-certification-kit-guide.md [hunk 2]
```diff
@@ -639,27 +639,27 @@ Examples:
...
 ```
-   # python ack_cli.py -n network.conf –o \
+   # ./ack_cli.py -n network.conf –o \
    "run_classes=operations_tests.CrashDumpTestClass"  
 ```
 2.Run only Multicast test class
 
 ```
-   # python ack_cli.py -n network.conf –o \
+   # ./ack_cli.py -n network.conf –o \
```
Comment (stephenchengCloud) [inline docs/xenserver-automated-certification-kit-guide.md:648]: This " –o" seems different from that in line 654 "-o"
One is long "–“， one is short "-". 
Please check.

File: docs/xenserver-automated-certification-kit-guide.md [hunk 3]
```diff
@@ -639,27 +639,27 @@ Examples:
 1.Run only Crash Dump test class
 
 ```
-   # python ack_cli.py -n network.conf –o \
+   # ./ack_cli.py -n network.conf –o \
    "run_classes=operations_tests.CrashDumpTestClass"
```
Comment (stephenchengCloud) [inline docs/xenserver-automated-certification-kit-guide.md:643]: Check "–o"

File: docs/xenserver-automated-certification-kit-guide.md (hunk 2 above)
Comment (catherineliuchina) [reply docs/xenserver-automated-certification-kit-guide.md]: I have updated line 642 and line 648 to short "-".

File: docs/xenserver-automated-certification-kit-guide.md (hunk 3 above)
Comment (catherineliuchina) [reply docs/xenserver-automated-certification-kit-guide.md:643]: I have updated line 642 and line 648 to short "-".

**Review by stephenchengCloud — APPROVED** [review]

**Review by xihuan-citrix — APPROVED** [review]

**Pull request merged**